### PR TITLE
zeroex - add native liquidity swaps to dex trades

### PR DESCRIPTION
**Pull request by RantumBits**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by RantumBits** [comment]
ready for review

File: models/zeroex/arbitrum/zeroex_arbitrum_native_fills.sql [hunk 1]
```diff
@@ -228,7 +228,7 @@ WITH
                 taker_token_filled_amount_raw as taker_token_amount_raw,
                 maker_symbol,
                 token_pair,
-                CAST(ARRAY() as array<bigint>) as trace_address,
+                '1' as trace_address,
```
Comment (jeff-dude) [inline models/zeroex/arbitrum/zeroex_arbitrum_native_fills.sql:231]: we had been defaulting this value to empty string, `""`, but we will be deprecating this field in the `dex.trades` redesign in the near future as it's not used across any projects added so far. with that in mind, i think it's okay to leave this as-is and let it go

**Review by jeff-dude — APPROVED** [review]
interesting, so you guys have two different products, one which can load into `dex.trades` and one that can load into `dex_aggregator.trades`?

**Comment by RantumBits** [comment]
> interesting, so you guys have two different products, one which can load into `dex.trades` and one that can load into `dex_aggregator.trades`?

yeah - dex.trades includes native liquidity trades (OTC, RFQ, and private-liquidity pools) while aggregator includes trades from other liquidity providers

**Pull request merged**